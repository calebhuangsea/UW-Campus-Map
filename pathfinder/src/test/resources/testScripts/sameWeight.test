# test a graph has two equal weight path

CreateGraph graph1

AddNode graph1 n1
AddNode graph1 n2
AddNode graph1 n3
AddNode graph1 n4

AddEdge graph1 n1 n2 1.000
AddEdge graph1 n1 n3 2.000
AddEdge graph1 n2 n4 2.000
AddEdge graph1 n3 n4 1.000

FindPath graph1 n1 n4