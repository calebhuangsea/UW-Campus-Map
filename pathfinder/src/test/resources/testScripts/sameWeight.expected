# test a graph has two equal weight path

created graph graph1

added node n1 to graph1
added node n2 to graph1
added node n3 to graph1
added node n4 to graph1

added edge 1.000 from n1 to n2 in graph1
added edge 2.000 from n1 to n3 in graph1
added edge 2.000 from n2 to n4 in graph1
added edge 1.000 from n3 to n4 in graph1

path from n1 to n4:
n1 to n2 with weight 1.000
n2 to n4 with weight 2.000
total cost: 3.000